src/Lean/Data/Lsp/Capabilities.lean

```diff
@@ -29,8 +29,17 @@ structure TextDocumentClientCapabilities where
   completion? : Option CompletionClientCapabilities
   deriving ToJson, FromJson
 
+structure ShowDocumentClientCapabilities where
+  support : Bool
+  deriving ToJson, FromJson
+
+structure WindowClientCapabilities where
+  showDocument? : Option ShowDocumentClientCapabilities
+  deriving ToJson, FromJson
+
 structure ClientCapabilities where
   textDocument? : Option TextDocumentClientCapabilities
+  window? : Option WindowClientCapabilities
```

Comment:
I can't find where this new "window" capability is used anywhere in the code?

Reply:
Yep, see the conversation above. It is however just part of the LSP spec, so adding it doesn't hurt.